src/lib/permission-types.ts

```diff
@@ -46,6 +46,7 @@ type Abilities =
     | Ability<'OrgMembers', 'view', { orgId?: UUID; orgSlug?: string }>
     | Ability<'Orgs', 'view', object>
     | Ability<'MFA', 'reset', object>
+    | Ability<'IDE', 'load', object>
```

Comment:
when speccing the type, it should require a object that matches the study shape:
```suggestion
    | Ability<'IDE', 'load', { researcherId: UUID }>
```